hokusai/production.yml

```diff
@@ -21,13 +20,33 @@ spec:
     spec:
       containers:
         - name: aprb-web
+          env:
+            - name: PORT
+              value: "4000"
           envFrom:
-          - configMapRef:
-              name: aprb-environment
+            - configMapRef:
+                name: aprb-environment
           image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/aprb:production
           imagePullPolicy: Always
           ports:
-          - containerPort: 4000
+            - name: aprb-http
+              containerPort: 4000
+          readinessProbe:
+            httpGet:
+              port: aprb-http
+              path: /ping
```

Comment:
This may actually be `/api/ping` @ashkan18 can you confirm?

Reply:
no, `ping` is correct 🙌 currently http://aprb.artsy.systems/ping